index.php

```diff
@@ -27,70 +27,83 @@
                                     <legend>Your documents</legend>
                                     <div class="form-row">
-                                        <label for="date_of_visit">Date of visit <span class="mandatory">&#42;</span></label>
-                                        <input type="text" id="date_of_visit" name="date_of_visit">
+                                        <p id="dob"><b>Date of visit</b></p>
+                                        <span id="reference-1-error" class="form-error form-hint">Please insert a valid day</span>
+                                        <div id="groupErrorPlacement">
+                                            <span class="dob-col day">
+                                                <label id="day" for="certificate_day">Day</label>
+                                                <input type="number" aria-labelledby="dob day" id="certificate_day" name="certificate-day" class="form-warning">
+                                            </span>
+                                            <span class="dob-col month">
+                                                <label id="month" for="certificate_month">Month</label>
+                                                <input type="number" aria-labelledby="dob month" id="certificate_month" name="certificate-month">
+                                            </span>
+                                            <span class="dob-col year">
+                                                <label id="year" for="certificate_year">Year</label>
+                                                <input type="number" aria-labelledby="dob year" id="certificate_year" name="certificate-year">
+                                            </span>
+                                        </div>
                                     </div>
                                     <div class="form-row">
                                         <p class="form-hint">Please enter your catalogue references in the fields below. To search the catalogue from this form, please click on the icon next to each field.</p>
                                     </div>
                                     <div class="form-row">
-                                        <label for="reference_1">Document 1 <span class="mandatory">&#42;</span></span></label>
-                                        <input type="text" id="reference_1" name="reference_1" class="medium form-warning"><a href=""><span class="scope-selector">&nbsp;</span></a>
+                                        <label for="reference_1">Document 1</label>
                                         <span id="reference-1-error" class="form-error form-hint">Please insert a reference number</span>
+                                        <input type="text" id="reference_1" name="reference_1" class="medium form-warning"><a href=""><span class="scope-selector">&nbsp;</span></a>
```

Comment:
for each of these inputs, name atts should be unique... but I know this is a prototype and will be defined alter... so not really an issue